selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/hotupdate/rolling/RollingUpdateStrategyWithWorkspacesStartStopTest.java

```diff
@@ -63,7 +51,7 @@ public void createMavenArchetypeStartProjectByWizard() throws Exception {
     workspaces.waitWorkspaceStatus(workspaceForStarting.getName(), Workspaces.Status.STOPPED);
 
     // execute rolling update command
```

Comment:
You may mean `// execute rollout update command`.

Reply:
Even more, I think "hotUpdateUtil.executeMasterPodUpdateCommand();" definitely clear command so comment is redundant here.